app/commands.py

```diff
@@ -1635,17 +1660,18 @@ async def mp_rmref(ctx: Context, match: Match) -> Optional[str]:
     if len(ctx.args) != 1:
         return "Invalid syntax: !mp addref <name>"
 
-    if not (t := app.state.sessions.players.get(name=ctx.args[0])):
+    target = app.state.sessions.players.get(name=ctx.args[0])
+    if not target:
         return "Could not find a user by that name."
 
-    if t not in match.refs:
-        return f"{t} is not a match referee!"
+    if target not in match.refs:
+        return f"{target} is not a match referee!"
 
-    if t is match.host:
+    if target is match.host:
         return "The host is always a referee!"
 
-    match._refs.remove(t)
-    return f"{t.name} removed from match referees."
+    match._refs.remove(target)
+    return f"{target.name} removed from match referees."
```

Comment:
seems this is a repetitive thing.. maybe best to keep for another pr

Reply:
`target.name` doesn't seem to be used that many times